packages/app/src/settings/device/NotificationSettings.vue

```diff
@@ -6,80 +6,114 @@
     </template>
-    <div class="divide-y border rounded divide-gray-50 border-gray-100 px-[16px]">
+    <div class="border rounded border-gray-100">
       <div
-        v-for="({id, title}) in switches"
-        :key="id"
-        class="py-[16px]"
+        v-if="!props.gql.localSettings.preferences.desktopNotificationsEnabled"
+        class="min-h-[56px] bg-indigo-50 px-[16px] py-[10px] flex flex-wrap justify-between items-center gap-[5px]"
+        data-cy="enable-notifications"
       >
-        <h4 class="flex text-gray-800 text-[16px] leading-[24px] items-center">
-          <label :for="id">{{ title }}</label>
-          <Switch
-            class="mx-[8px]"
-            :value="props.gql.localSettings?.preferences[id] ?? false"
-            :name="id"
-            @update="(value) => updatePref(value, id)"
+        <div class="text-indigo-700 font-medium flex items-center">
+          <IconSecurityLockLocked
+            class="mr-[7px]"
+            fill-color="indigo-200"
+            stroke-color="indigo-500"
           />
-          {{ gql.localSettings }}
-        </h4>
-      </div>
-      <div class="py-[16px]">
-        <h4 class="flex text-gray-800 text-[16px] leading-[24px] items-center">
-          {{ t('settingsPage.notifications.notifyMeWhenRunCompletes') }}
-        </h4>
-        <div class="pt-[5px]">
-          <Checkbox
-            v-for="({id, label}) in statuses"
-            :id="id"
-            :key="id"
-            v-model="listRef"
-          >
-            <template #label>
-              <label
-                :for="id"
-                class="text-sm text-gray-900"
-              >
-                {{ label }}
-              </label>
-            </template>
-          </Checkbox>
+          {{ t('settingsPage.notifications.enableNotificationsLabel') }}
         </div>
-      </div>
-      <div class="py-[16px]">
-        <Button
-          variant="white"
+
+        <ButtonDS
           size="32"
-          @click="showTestNotification"
+          class="font-normal"
+          @click="enableNotifications"
         >
-          {{ t('settingsPage.notifications.sendTestNotification') }}
-        </Button>
-        <i18n-t
-          tag="p"
-          scope="global"
-          keypath="settingsPage.notifications.notReceivingNotifications"
-          class="pt-[15px] text-gray-600 text-sm font-normal"
+          {{ t('specPage.banners.enableNotifications.enableDesktopNotifications') }}
+        </ButtonDS>
+      </div>
+
+      <div class="px-[16px] divide-y divide-gray-50">
+        <div
+          v-for="({id, title}) in switches"
+          :key="id"
+          class="py-[16px]"
         >
-          <ExternalLink :href="troubleshootingHref">
-            {{ t('settingsPage.notifications.troubleshoot') }}
-          </ExternalLink>
-        </i18n-t>
+          <h4 class="flex text-gray-800 text-[16px] leading-[24px] items-center">
+            <label :for="id">{{ title }}</label>
+            <Switch
```

Comment:
The switches have a funny border flash when hovered over. This is probably an issue with the Switch component, but thought I would point it out.

https://github.com/cypress-io/cypress/assets/1702361/d92da477-5097-4dd2-a4e4-664554d74471

Reply:
Ah, missed this one. Trying not too put too much work into the design components here, since that really should go in the design system, but this is probably a quick fix.

Reply:
Fixed this one.